### PR TITLE
created new spinner page

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5388

*Description of changes:*
- created new Search Pending component
- display new component on Find Name Request
- misc cleanup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
The new Search Pending component looks like this:

SearchPending

File: client/src/store/new-request-module.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import ExistingRequestEdit from '@/components/existing-request/existing-request-edit.vue'
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:1]: Wasn't used.

File: client/src/views/landing.vue [hunk 2]
```diff
@@ -24,14 +24,14 @@ import AnalyzeCharacters from '@/components/new-request/analyze-characters.vue'
 import AnalyzePending from '@/components/new-request/analyze-pending.vue'
 import AnalyzeResults from '@/components/new-request/analyze-results.vue'
 import ExistingRequestDisplay from '@/components/existing-request/existing-request-display.vue'
-import Success from '@/components/common/success.vue'
+import ExistingRequestEdit from '@/components/existing-request/existing-request-edit.vue'
 import LowerContainer from '@/components/lower-info-area/lower-container.vue'
-import newRequestModule from '@/store/new-request-module'
+import SearchPending from '@/components/existing-request/search-pending.vue'
```
Comment (severinbeauvais) [inline client/src/views/landing.vue:29]: I added this line and then alphabetized the imports :)

**Review by scottrumsby — APPROVED** [review]
looks good to mw

File: client/src/components/existing-request/search-pending.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <v-col cols="auto">
+          <b>Please wait a moment</b>
+        </v-col>
+      </v-row>
+      <v-row justify="center">
+        <v-col cols="auto">
+          <v-progress-circular color="orange"
+                               id="analyze-pending-spinner"
+                               size="50"
+                               indeterminate />
```
Comment (cameron-freshworks) [inline client/src/components/existing-request/search-pending.vue:20]: Spacing seems a bit off here.

File: client/src/components/existing-request/search-pending.vue [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+</template>
+
+<script lang="ts">
+import MainContainer from '@/components/new-request/main-container.vue'
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({
+  components: { MainContainer }
+})
+export default class SearchPending extends Vue {}
```
Comment (cameron-freshworks) [inline client/src/components/existing-request/search-pending.vue:39]: Impressive! 1 line component 😁

File: client/src/components/new-request/analyze-pending.vue [hunk 5]
```diff
@@ -11,31 +11,31 @@
...
+        <v-col cols="auto">
+          <b>This could take several minutes</b>
+        </v-col>
+      </v-row>
+      <v-row justify="center">
+        <v-col cols="auto">
+          <v-progress-circular color="orange"
+                               id="analyze-pending-spinner"
+                               size="50"
+                               indeterminate />
```
Comment (cameron-freshworks) [inline client/src/components/new-request/analyze-pending.vue:23]: Maybe spacing isn't off? Is this standard?

**Review by cameron-freshworks — APPROVED** [review]
Looks good! Just had minor a question/comment

File: client/src/components/existing-request/search-pending.vue (hunk 3 above)
Comment (severinbeauvais) [reply client/src/components/existing-request/search-pending.vue:20]: This code indentation isn't my favourite style but I kept it consistent with the file I copied this from (analyze-pending.vue).
Is this what you are referring to, and do you think I should change it? I don't mind.

File: client/src/components/new-request/analyze-pending.vue (hunk 5 above)
Comment (severinbeauvais) [reply client/src/components/new-request/analyze-pending.vue:23]: It's the existing code style. I'm happy to leave it alone for now 😁

File: client/src/components/existing-request/search-pending.vue (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/components/existing-request/search-pending.vue:20]: oh yea, i'm totally fine if thats uniform in this app :)

File: client/src/components/new-request/analyze-pending.vue (hunk 5 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/analyze-pending.vue:23]: haha totally, works for me!

**Pull request merged**